src/main/java/com/spring/onlinebookstore/controller/AuthController.java

```diff
@@ -0,0 +1,28 @@
+package com.spring.onlinebookstore.controller;
+
+import com.spring.onlinebookstore.dto.user.UserRegistrationRequestDto;
+import com.spring.onlinebookstore.dto.user.UserResponse;
+import com.spring.onlinebookstore.exception.RegistrationException;
+import com.spring.onlinebookstore.security.AuthenticationService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/auth")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

add `server.servlet.context-path=/api` to application properties